src/font.rs

```diff
@@ -107,12 +107,121 @@ impl<'a> Stream<'a> {
     }
 }
 
+/// The values of a [`font` shorthand](https://www.w3.org/TR/css-fonts-3/#font-prop).
```

Comment:
So this is CSS3 `font` shorthand, not SVG 1 `font` one? Do they differ much?

Reply:
Yes, it's the CSS3 one. Seems like the only difference is that the SVG 1 one doesn't support the `font-stretch` attribute.